README.md

```diff
@@ -57,7 +57,7 @@ _You can use any library you prefer: Ethers.js, Web3.js, Web3.py, eth-brownie_
 
 Web3 projects often setup the governance process for protocol changes via a combination of a DAO, core contributors, community members and token holders. This means that any core contributor must first propose a change, share with the public, and then the proposal gets more validation as stakeholders spend time discussing the pros and cons.
 
-Create an Improvement Proposal by describing the work in Challenge #1. You can add any other relevant information that could help the community to decide
+Create an Improvement Proposal by either describing the work in Challenge #1 or proposing a new feature/modification to it. You can add any other relevant information that could help the community to decide
```

Comment:
While we're doing this, it might be worth noting that the proposal should go as a markdown file in the solution repo, rather than actually posting it in the forum